content/docs/09-registries-and-packs/1-adding-a-custom-registry.md

```diff
@@ -36,10 +36,61 @@ Pack contents are periodically synchronized with Palette.
 Please ensure that the ports 443 and 80 are exclusively allocated to the registry server and are not in use by other processes.
 </WarningBox>
 
-# Deploying a Pack Registry Server
-
-Palette provides a Docker image for the pack registry server. The following steps need to be performed to deploy the pack registry server using this docker image:
-
+# Deploying a Pack Registry Server with letsencrypt certificates
```

Comment:
```suggestion
# Deploy a Pack Registry Server with SSL Certificates
```
Just trying to think of ways to shorten the title.

Reply:
Adapted a bit. Hope the new title is better.